index.html

```diff
@@ -54,145 +56,181 @@
-                  <center>
-                    <div class="progress" style="width:80%">
-                      <div class="progress-bar progress-bar-success progress-bar-striped" role="progressbar" aria-valuenow="12" aria-valuemin="0" aria-valuemax="100" style="width: 12%;"></div>
-                    </div>
-                  </center>
-                </button>
-              </div>
+            <div class="col-md-3 col-sm-6">
+              <button type="button" class="btn mybutton" onclick="location.href='learn.html';">
+                <h3>DataSetName2</h3>
+
+                <br>
+                <h4>Strength:</h4>
+                <center>
+                  <div class="progress" style="width:80%">
+                    <div class="progress-bar progress-bar-success progress-bar-striped" role="progressbar" aria-valuenow="68" aria-valuemin="0" aria-valuemax="100" style="width: 68%;"></div>
+                  </div>
+                </center>
+              </button>
             </div>
 
-            <div class="row" style="margin-bottom:50px">
-              <div class="col-md-4">
-                <button type="button" class="btn btn-danger" style="width:100%;height:200px">
-                  <h3>DataSetName7</h3>
-
-                  <br>
-                  <h4>Strength:</h4>
-                  <center>
-                    <div class="progress" style="width:80%">
-                      <div class="progress-bar progress-bar-success progress-bar-striped" role="progressbar" aria-valuenow="99" aria-valuemin="0" aria-valuemax="100" style="width: 99%;"></div>
-                    </div>
-                  </center>
-                </button>
-              </div>
-
-              <div class="col-md-4">
-                <button type="button" class="btn btn-danger" style="width:100%;height:200px">
-                  <h3>DataSetName8</h3>
-
-                  <br>
-                  <h4>Strength:</h4>
-                  <center>
-                    <div class="progress" style="width:80%">
-                      <div class="progress-bar progress-bar-success progress-bar-striped" role="progressbar" aria-valuenow="21" aria-valuemin="0" aria-valuemax="100" style="width: 21%;"></div>
-                    </div>
-                  </center>
-                </button>
-              </div>
-
-              <div class="col-md-4">
-                <button type="button" class="btn btn-danger" style="width:100%;height:200px">
-                  <h3>DataSetName9</h3>
-
-                  <br>
-                  <h4>Strength:</h4>
-                  <center>
-                    <div class="progress" style="width:80%">
-                      <div class="progress-bar progress-bar-success progress-bar-striped" role="progressbar" aria-valuenow="39" aria-valuemin="0" aria-valuemax="100" style="width: 39%;"></div>
-                    </div>
-                  </center>
-                </button>
-              </div>
+            <div class="col-md-3 col-sm-6">
```

Comment:
I know I've already said this, but build this list with code, don't hardcode it.